lib/escalas/result_questions.dart

```diff
@@ -0,0 +1,233 @@
+import 'package:app_mental/escalas/recommendation_dialog.dart';
+import 'package:app_mental/escalas/success_dialog.dart';
+import 'package:app_mental/helper/constants.dart';
+import 'package:flutter/material.dart';
+
+import '../Services/questionnaireService.dart';
+import 'critical_dialog.dart';
+
+class ResultQuestions extends StatefulWidget {
+  final String questionnaireCode;
+  final String userEmail;
+  final String questName;
+  final String userEscala;
+
+  ResultQuestions({
+    required this.questionnaireCode,
+    required this.userEmail,
+    required this.questName,
+    required this.userEscala,
+  });
+
+  @override
+  State<ResultQuestions> createState() => _ResultQuestionsState();
+}
+
+class _ResultQuestionsState extends State<ResultQuestions> {
+  bool hasRecommendation = false;
+  int score = 0;
+  List<int> scoreList = [];
+
+  final String resultPhrase =
+      'Questionário concluído! \n\nSuas respostas serão enviadas, e analisadas anonimamente para a recomendação de novas atividades.\n\nEstá de acordo?';
+
+  getScore() async {
+    int sum = 0;
+    await QuestionnaireService()
+        .getScore(widget.userEmail, widget.questionnaireCode, widget.userEscala)
+        .then((values) {
+      for (var index = 5; index < values.length; index++) {
+        sum += int.parse(values[index]);
+      }
+    });
+    setState(() {
+      score = sum;
+    });
+  }
+
+  getScoreList() async {
+    await QuestionnaireService()
+        .getScore(widget.userEmail, widget.questionnaireCode, widget.userEscala)
+        .then((values) {
+      values.forEach((value) {
+        scoreList.add(int.parse(value));
+      });
+    });
+  }
+
+  verifyScore() {
+    if (QuestionnaireCode.psqi.name == widget.questionnaireCode &&
+        score > Constants.recommendationValuePsqi) {
+      /*Map<String, dynamic> readingsMap1 = {
```

Comment:
Pode remover o comentario?

Reply:
Esse eu já falei pra vc lá em cima. Ele está com marcação para quando for fazer a notificação dele. Sem isso não sei quais são as leituras